### ISSUE TITLE
No props table appear within docs on typescript when exported from another file

### ISSUE THREAD

**Issue by sney-js** [storybookjs/storybook#10610]
**Describe the bug**
It seems like unless the types are in the same file as the Component, it is not shown within `@addons/docs` prop section.

**To Reproduce**
The code below for a react component **SHOWS** props within the docs section.
`Button.tsx`: 
```
...

type ButtonProps = {
  title?: string;
  appearance?: 'primary' | 'inverse' | 'invisible';
};

const Button: FC<ButtonProps> = (props: ButtonProps) => {
...
}
```
`Button.stories.tsx`:
```
import React from 'react';
import Button from './Button';

export default {
  title: 'Elements|Button',
  component: Button
};

const Basic = () => <Button title="test" />
```
But if I move the ButtonProps type to another file, types table does not work:
```
...
import { ButtonProps } from './ButtonModels';
const Button: FC<ButtonProps> = (props: ButtonProps) => {
...
}
```

**Expected behavior**
Regardless of where the types are exported from, types table should show?

**Screenshots**
![image](https://user-images.githubusercontent.com/6628037/80806589-f6d7d180-8bb2-11ea-9976-eb3a0c868917.png)
When types exported:
![image](https://user-images.githubusercontent.com/6628037/80806603-0c4cfb80-8bb3-11ea-9e3e-530b371c2726.png)

**Comment by shilman** [comment]
Documented here: https://github.com/storybookjs/storybook/blob/next/addons/docs/docs/props-tables.md#imported-types

**Comment by shilman** [comment]
A few weeks ago we released zero-config typescript support in 6.0-beta, and then a few days ago upgraded `react-docgen-typescript-loader` to `react-docgen-typescript-plugin` in the latest beta. This will solve many typescript props-related issues, possibly including this one:

- [Migration instructions](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#zero-config-typescript)
- [Updated docs](https://github.com/storybookjs/storybook/blob/next/docs/src/pages/configurations/typescript-config/index.md)

Please try it out!
